HouseRules_Essentials/README.md

```diff
@@ -507,7 +554,8 @@ The [Settings Reference](../docs/SettingsReference.md) contains lists of all dif
   - Max of two replenishable cards per player.
   - Config accepts Dictionary of list of dicts e.g. `{ "HeroName1": [ { "Card" : "CardName","isReplenishable": bool }, ... ], ...  }`
 
-  ###### _Example JSON config for StartCardsModifiedRule_
+  <details>
```

Comment:
If we are changing to use <details> blocks in the docs, it would be good to ensure that we caputre all of the rules. The more recently added rules for AbilityBackstabAdjusted, BackStabConfig, RegainAbilityifMaxxedOut and  CardClassRestrictionOverriddenRule still have their JSON examples up-front. My personal prefrence is to have the JSON displayed without any extra clicking, but as I suffer from 'interface designed by an engineer syndrome', I feel that Mex should make the final call on the format of the docs.

Reply:
Ugh. I think I'm making it worse not better. By deleting the file from this branch does that mean it will remove it from main?